lesson_23/script.js

```diff
@@ -0,0 +1,24 @@
+let array = [12, 25, 6, 48, 33, 17, 9, 56, 40, 22];
```

Comment:
you should add some boolean, string and null values to check that your code works correct and ignore not number values